### DISCUSSION TITLE
Beating score with new mod score multipliers doesn't overwrite

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#19353] [Q&A] [answered]
In the new lazer update, difficulty increasing mods that have custom settings have had their score multipliers decreased to 1.00x temporarily. However, when I try to submit a score that would be higher in this version of lazer but lower in the previous version of lazer, it will not overwrite my previous score. I know that the score will submit since it shows up on the selected mods leaderboard, but it will not overwrite my personal best despite it being a higher score.

For example, what should now be my highest score (used to be ~1.27 million and stayed the same):
![image](https://user-images.githubusercontent.com/74088315/180660102-53c0678d-9f88-4235-9aa5-f4ea4dd2a634.png)

What actually shows up as my personal best (used to be ~1.33 million, is now 1 million):
![image](https://user-images.githubusercontent.com/74088315/180660114-f73ae5ea-c0b3-47a2-9074-f10613507634.png)

Basically as it stands, my personal best is unable to be overwritten until I get a score that is greater than ~1.33 million despite it having been reduced to 1 million.

**Comment by peppy** [comment, marked as answer]
This will be fixed server side at a later point.
